README.md

```diff
@@ -147,6 +147,31 @@ Note:
 - `--p2p-discovery-bootnodes` for ENR list (comma separated), if working with discovery, public net etc.
 - `--Xdata-storage-non-canonical-blocks-enabled` is added to store non-canonical blocks for debugging purposes.
 
+##### Docker
+
+Use the following command to run Teku from docker:
+```shell
+docker -v ./$TESTNET_NAME:/testnet -p 9000:9000/tcp -p 9000:9000/udp -p 8008:8008 -p 5051:5051 \
```

Comment:
We may just want `--net host` instead, but this should work too.

Reply:
yes, It may also be useful for eth1 endpoint configuration for the case when eth1 client is running on the host